src/test/kotlin/graphics/scenery/tests/examples/basic/TexturedCubeExample.kt

```diff
@@ -52,6 +52,11 @@ class TexturedCubeExample : SceneryBase("TexturedCubeExample") {
                 Thread.sleep(20)
             }
         }
+
+        thread {
+            Thread.sleep(5000)
+            System.gc()
```

Comment:
leftover from testing?

Reply:
Indeed, that is totally not supposed to be there. Good catch!